frontend/public/components/utils/resource-log.jsx

```diff
@@ -247,17 +283,21 @@ export class ResourceLog extends SafetyFirst {
           Refresh
         </button>
       </p>}
-      <LogControls
-        dropdown={dropdown}
-        status={status}
-        toggleStreaming={this._toggleStreaming}
-        onDownload={this._download} />
-      <LogWindow
-        lines={lines}
-        linesBehind={linesBehind}
-        bufferFull={bufferFull}
-        status={status}
-        updateStatus={this._updateStatus} />
+      <div ref={this._logConsole} className={classNames('log-console', {'log-console--toolbar__fullscreen': isFullscreen})}>
```

Comment:
Per BEM, this class would need to go on the toolbar element itself

Reply:
Also per @TheRealJon it should be `log-console__toolbar--fullscreen`. (I got the BEM naming wrong.)